src/librustdoc/clean/auto_trait.rs

```diff
@@ -471,9 +459,7 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
         let mut ty_to_fn: FxHashMap<Type, (Option<PolyTrait>, Option<Type>)> = Default::default();
 
         for p in clean_where_predicates {
-            let replaced = p.fold_with(&mut replacer);
-            let (orig_p, p) = (replaced, replaced.clean(cx));
-
+            let (orig_p, p) = (p, p.clean(self.cx));
```

Comment:
This seems like a behavior change.

Reply:
It's not actually - it's just that the predicate is folded above (in `clean_where_predicates`) instead of here. It looks confusing because you're only looking at 2 commits, not the full diff.